include/swift/AST/AttrKind.h

```diff
@@ -130,27 +130,27 @@ enum class ExternKind: uint8_t {
 enum : unsigned { NumExternKindBits =
   countBitsUsed(static_cast<unsigned>(ExternKind::Last_ExternKind)) };
 
-enum DeclAttrKind : unsigned {
-#define DECL_ATTR(_, NAME, ...) DAK_##NAME,
-#include "swift/AST/Attr.def"
-  DAK_Count
+enum class DeclAttrKind : unsigned {
```

Comment:
Do you know if there's any reason DeclAttrKind has to be unsigned? If it's just for DAK_Count, NumDeclAttrKinds covers that now. Doesn't really matter either way though so 🤷 (just pointing it out since it seemed like part of the reason for this PR was to mostly "unify" decl/type attr kinds)